google/cloud/forseti/scanner/audit/kms_rules_engine.py

```diff
@@ -384,9 +383,9 @@ def find_violations(self, key):
         """
         violations = []
         creation_time = key.primary_version.get('createTime')
+        state = key.primary_version.get('state')
         scan_time = date_time.get_utc_now_datetime()
-
-        if self.rule['mode'] == BLACKLIST:
+        if self.rule['mode'] == BLACKLIST and state == 'ENABLED':
```

Comment:
Since we might plan to support other modes in the future, can we separate the 2 checks in this if statement?

e.g. put the following under "state = key.primary_version.get('state')"

if state != 'ENABLED':
  // We don't scan for keys that are not currently enabled.
  return []